maintainers/maintainer-list.nix

```diff
@@ -8919,6 +8919,16 @@
     githubId = 687198;
     name = "Yuri Aisaka";
   };
+  yurkobb = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add yurkobb``.